modules/acquia_cms_common/src/Services/ConfigImporterService.php

```diff
@@ -15,6 +15,7 @@
 use Drupal\Core\Extension\ThemeHandlerInterface;
 use Drupal\Core\Lock\LockBackendInterface;
 use Drupal\Core\StringTranslation\TranslationInterface;
+use Drupal\Core\Utility\Error;
```

Comment:
Does this class available in below D10.1 version?

Reply:
yes it is available in 10.0